setup.py

```diff
@@ -143,7 +141,7 @@ def run_tests(self):
 
 setup(
     name='openhtf',
-    version='1.0',
+    version='1.1',
```

Comment:
we have releases happening in that i am building sdist tarballs and have them lying around, so the distinction between being installable on windows and not is somewhat important to me.  what's the cost of incrementing this?